test/vdom.test.js

```diff
@@ -0,0 +1,475 @@
+import { h, patch } from "../src"
+
+window.requestAnimationFrame = setTimeout
```

Comment:
I don't think any code in picodom uses requestAnimationFrame (it is used in hyperapp though). Perhaps I should remove this line altogether since it doesn't do anything to affect these tests?